lya/data.py

```diff
@@ -40,10 +41,30 @@ def plot_pk(params=None, filename=lya_data_path):
 
 
 def get_data_transfer_function():
+    
+   
     """Returns the data "transfer function" 
 
     Returns
     -------
     data : Data object
         Data object representing the ly-a data divided by LCDM p(k)
     """
+    
+    # 
+    pkncdm_data = load_lya_data()
```

Comment:
does 'ncdm' here refer to "non-cold dm"?  I think I'd prefer something simpler; how about just calling it `data`?